hummingbot/client/ui/style.py

```diff
@@ -120,6 +120,8 @@ def hex_to_ansi(color_hex):
     "primary":                    "#1CD085",  # noqa: E241
     "warning":                    "#93C36D",  # noqa: E241
     "error":                      "#F5634A",  # noqa: E241
+    "tab_button.focused":         "bg:#1CD085 #000000",  # noqa: E241
```

Comment:
I don't think this belongs here. hex_to_ansi() is just for converting individual color values from hex to windows ANSI codes because Windows command line is very limited. Styles of specific components are defined on the bottom of this file.

Reply:
... and the top functions apply the configurable settings to the styles. The reason for this is that there's no 1-to-1 correspondence between user defined settings and the actual styles of the components, and at the same time the user defined settings don't represent just simple colors, they're like styles applied to the actual styles, creating "connections" between e.g. foreground colors of multiple toolkit styles. You should define the base style of the component on the bottom and then in the top functions define how will the user settings affect it, and use the hex_to_ansi() function to convert the individual colors in hex chosen by the user for the particular OS.